autogpts/forge/forge/command/command.py

```diff
@@ -3,11 +3,9 @@
 import inspect
 from typing import Any, Callable
 
-from .command_parameter import CommandParameter
-from .context_item import ContextItem
+from .parameter import CommandParameter
 
-CommandReturnValue = Any
-CommandOutput = CommandReturnValue | tuple[CommandReturnValue, ContextItem]
+CommandOutput = Any
```

Comment:
Why this change? won't that break typehinting on commands?

Reply:
`ContextItem`s are handled by `ContextComponent` and so the tuple is redundant:
```
CommandReturnValue = Any
CommandOutput = CommandReturnValue
```
which is the same as `CommandOutput = Any`
I don't think that's any problem for type hinting.